### PR TITLE
Report CoT upload errors to task log

### PR DESCRIPTION
**Pull request by walac**

### CODE REVIEW

**Comment by walac** [comment]
Retriggering the failed job and it worked.

File: src/lib/features/chain_of_trust.js [hunk 1]
```diff
@@ -65,7 +66,7 @@ class ChainOfTrust {
         });
       await fs.unlink(this.file.path);
     } catch (err) {
-      debug(err);
+      task.stream.write(fmtErrorLog(err));
```
Comment (djmitche) [inline src/lib/features/chain_of_trust.js]: This writes tot the task log, right?

Comment (walac) [reply src/lib/features/chain_of_trust.js]: Oops, you are right. Don't know why I wrote papertrail, let me change the commit message.

**Comment by djmitche** [comment]
Awesome, visibility in the task log will help a lot!

**Pull request merged**